payjoin-relay/tests/integration.rs

```diff
@@ -356,10 +360,15 @@ mod integration {
 
     // Receiver receive and process original_psbt from a sender
     // In production it it will come in as an HTTP request (over ssl or onion)
-    fn handle_relay_response(res: Vec<u8>, receiver: bitcoincore_rpc::Client) -> String {
-        // Receiver receive payjoin proposal, IRL it will be an HTTP request (over ssl or onion)
+    fn handle_relay_response(
+        res: Vec<u8>,
+        receiver: bitcoincore_rpc::Client,
+        context: ProposalContext,
+    ) -> PayjoinProposal {
```

Comment:
this is changed significantly from the source `payjoin` integration test. That adds complexity